README.MD

```diff
@@ -1,4 +1,5 @@
 # tsoa
+Pronounced so·uh
```

Comment:
You’re welcome to make this change to whatever you’d like since you’re the founder, but may I put forward “tee-ess-so-uh”? I always like that it sounds like service oriented architecture (SOA) for TypeScript haha.